src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.html

```diff
@@ -2,42 +2,59 @@
   <title>Error Page</title>
   <base target="_blank">
 </head>
-<div class="dashboard row">
-  <div class="text-center content">
-    <br>
-    <div *ngIf="header && message; else elseBlock">
-    <i class="{{ icon }}"
-       aria-hidden="true"></i>
-    <br><br><br>
-    <h3><b>{{ header }}</b></h3>
-      <br>
-    <h4>{{ message }}</h4>
+<div class="container h-75">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <div *ngIf="header && message; else elseBlock">
+        <i [ngClass]="icon"
+          class="mx-auto d-block"></i>
+
+        <div class="mt-4 text-center">
+          <h3><b>{{ header }}</b></h3>
+          <h4 class="mt-3"
+              *ngIf="header !== message">{{ message }}</h4>
+            <h4 *ngIf="section"
+                i18n>Please consult the <a href="{{ docUrl }}">documentation</a> on how to configure and enable
+                     the {{ sectionInfo }} management functionality.</h4>
+        </div>
+      </div>
+
+      <div class="mt-4">
+        <div class="text-center"
+             *ngIf="(buttonName && buttonRoute) || bootstrap; else dashboardButton">
+          <button class="btn btn-primary"
+                  [routerLink]="buttonRoute"
+                  *ngIf="buttonName && buttonRoute && !bootstrap"
+                  i18n>{{ buttonName }}</button>
+
+          <button class="btn btn-primary"
+                  (click)="doConfigure()"
+                  [attr.title]="buttonTitle"
+                  *ngIf="bootstrap"
```

Comment:
Instead of `bootstrap`, what about something that makes it easier to understand the different ways of "escaping" from an empty space:
* Configuring by visiting a UI form (`ui-config = true`?).
* Configuring by triggering a backend action (`backend-config = true`, `zero_config = true`?)

Reply:
went with `uiConfig`.